src/machine/machine_atsamd51.go

```diff
@@ -1882,7 +1883,7 @@ func (usbcdc *USBCDC) Flush() error {
 }
 
 // WriteByte writes a byte of data to the USB CDC interface.
-func (usbcdc *USBCDC) WriteByte(c byte) error {
+func (usbcdc USBCDC) WriteByte(c byte) error {
```

Comment:
Is there a specific reason you changed the signature here? I think we probably do want to keep it as a pointer receiver.

Reply:
No, there is no particular reason.
I just made it look the same as `machine_nrf52840_usb.go` .

I'll fix it.